data/decc/attributes.json

```diff
@@ -1,5 +1,5 @@
 {
-    "version": "20240208v1",
+    "version": "20240703v1",
```

Comment:
AGAGE folks requested that we remove the "v1" from the version numbers. I'll remove this